test/runner/lib/docker_util.py

```diff
@@ -185,6 +197,8 @@ def docker_inspect(args, container_id):
         stdout = docker_command(args, ['inspect', container_id], capture=True)[0]
         return json.loads(stdout)
     except SubprocessError as ex:
+        if 'no such image' in ex.stderr:
+            return {} # podman does not handle this gracefully, exits 125
```

Comment:
Ah, sorry, I should have looked at the type annotations instead of the code you'd already written.

The return should be `[]` instead of `{}`.

```suggestion
            return [] # podman does not handle this gracefully, exits 125
```